pkg/test/istio.io/tasks/security/mtls-migration/configure-mtls-destinationrule.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
what is the plan with these bash scripts?

Reply:
These are literally pulled from istio.io. Currently the examples don't reference anything externally. I'd love to see a point where we can link the ones in these tests to the istio.io ones.